Client/core/Graphics/CGraphics.cpp

```diff
@@ -1138,7 +1138,8 @@ void CGraphics::DrawStringQueued(float fLeft, float fTop, float fRight, float fB
             fRight = floor(fRight);
             fBottom = floor(fBottom);
         }
-
+        if (fLineHeight == 0.0f)
```

Comment:
It's better to use `std::optional` in here I think

Reply:
Okay, I see some of modules(game_sa or multiplayer_sa) do not support C++17 features. You can't use std::optional here.

Reply:
Yeah, apparently they've changed something in MSVC C++17 which caused naked functions to break.

Reply:
Btw I think he could use negative values to indicate something is wrong (as you obviously can't have negative spacing).

Reply:
> Btw I think he could use negative values to indicate something is wrong (as you obviously can't have negative spacing).

Well, you can(last example in screenshot below, where the last line became the first)